Correction to mc.cores argument for Windows compat (#1)
sgibb/bootstrap: README.md

```s
install.packages("devtools")
library("devtools")

> install_github("sgibb/bootstrap") #may be asked to DL Rtools for windows; no issues, just do it :)

Reviewer: Thanks for the fix. Could you please adapt the wording a little bit:
```suggestion
install_github("sgibb/bootstrap") # may be asked to download Rtools on windows
```

## plot
hc <- createHclustObject(USArrests)

> plot(hc) #Note add "hang=-1" for lining up the terminals

Reviewer: ```suggestion
plot(hc) # add "hang=-1" for lining up the terminal endings
```
